### DISCUSSION TITLE
Ekspand. tabeller skillelinje

### DISCUSSION THREAD

**Discussion by ivnai** [fremtind/jokul#3617] [Annet]
Hei, hva med å fjerne skillelinje i ekspanderende tabelle open?
<img width="920" alt="Screenshot 2023-07-10 at 11 49 36" src="https://github.com/fremtind/jokul/assets/93194579/7d74d85d-d9ee-43b7-9791-302c030fbf8f">

**Comment by Stakset** [comment]
Slik har vi det i PM
<img width="546" alt="PM" src="https://github.com/fremtind/jokul/assets/72978199/cb5d5f26-e165-4d3b-a18d-6ff023a34764">
